_data/docs.yml

```
@@ -144,6 +144,19 @@ apis:
       legacy: 0
       stable: 1
       nightly: 1
+  ucxx:
+    name: UCXX
+    path: ucxx
+    desc: 'UCXX is the Python interface for [UCX(<https://github.com/openucx/ucx), a low-level high-performance networking library. UCX and UCXX support several transport methods including InfiniBand and NVLink while still using traditional networking protocols like TCP.'
+    ghlink: https://github.com/rapidsai/ucxx
+    cllink: https://github.com/rapidsai/ucxx/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      # TODO: enable 'legacy' once it moves to 0.48, and add version to projects-to-versions.json
+      legacy: 0
+      # TODO: enable 'stable' once it moves to 0.47, and add version to projects-to-versions.json
```

```suggestion
      # TODO: enable 'legacy' once it moves to 0.46
      legacy: 0
      # TODO: enable 'stable' once it moves to 0.46
```

I don't think these comments are what you want. `0.46` will be the first version that has docs on S3, so you can enable these other version types once those point to 0.46.

This is configured here:

https://github.com/rapidsai/docs/blob/711b940853d7f91674000e36a3276efd239b7b6b/_data/releases.json#L2-L14

Once `stable` points to `0.46`, it'd be safe to set `stable: 1` for UCXX here.
Once `legacy` points to `0.46`, it'd be safe to set `legacy: 1` for UCXX here.

> _add version to projects-to-versions.json_

No need to document this, this is done automatically by a pre-commit hook that reads from `_data/docs.yml`:

https://github.com/rapidsai/docs/blob/711b940853d7f91674000e36a3276efd239b7b6b/.pre-commit-config.yaml#L38-L46

---

Ah ok, so the comment is relative to `_data/releases.json`, i.e., when `stable`/`legacy` points to `0.46` in `_data/releases.json`, not when we move to a new release. Understood now, thanks for clarifying.